health-sdk/sdk/src/test/java/net/gini/android/health/sdk/review/ReviewViewModelTest.kt

```diff
@@ -71,8 +72,11 @@ class ReviewViewModelTest {
     fun `shows info bar on launch`() = runTest {
```

Comment:
[Request] Please add a few test cases to check that the `popupDurationPaymentReview` variable is within the agreed interval (0-10). For example, testing negative numbers, and numbers higher than 10.